### PR TITLE
Functions Should Output PSCustomObjects #8

### PR DESCRIPTION
**Pull request by ghost**
Converted output to PScustomobject. In some functions the output was in the end{}, so I moved that block into the Process{}. For example in New-Hipchatroom

### CODE REVIEW

File: HipChatAdmin/Public/New-HipchatUser.ps1 [hunk 1]
```diff
@@ -82,19 +82,23 @@ function New-HipchatUser{
...
-
-	END {
-
 		# Check response status code #
 		if ($Call.StatusCode -eq '201') {
 			Write-Verbose "User Created Successfully!"
-			Write-Output $Call.StatusCode
+			$OutputObject = New-Object -TypeName PSObject
+			$OutputObject | Add-Member -MemberType 'NoteProperty' -Name 'MentionName' -Value $MentionName
+            $OutputObject | Add-Member -MemberType 'NoteProperty' -Name 'FirstName' -Value $FirstName
```
Comment (cofonseca) [inline HipChatAdmin/Public/New-HipchatUser.ps1]: Not sure if it's a GitHub display issue or not, but some of the indentation seems off on each of these changes. I'd like to keep white space and indentation consistent.

File: HipChatAdmin/Public/Add-HipchatUserToRoom.ps1 [hunk 2]
```diff
@@ -48,7 +48,10 @@ function Add-HipchatUserToRoom{
 			# Check response status code #
 			if ($Call.StatusCode -eq '204') {
 				Write-Verbose "User $Name Added Successfully!"
-				Write-Output $Call.StatusCode
+                $OutputObject = New-Object -TypeName PSObject
+				$OutputObject | Add-Member -MemberType 'NoteProperty' -Name 'Name' -Value $name
+				$OutputObject | Add-Member -MemberType 'NoteProperty' `-Name 'StatusCode' -Value $Call.StatusCode
```
Comment (cofonseca) [inline HipChatAdmin/Public/Add-HipchatUserToRoom.ps1]: Each one of these has a backtick between 'NoteProperty' and -Name. Is this intentional?

**Review by cofonseca — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
I have pushed two commits to fix the formatting and remove the backticks.

**Review by cofonseca — APPROVED** [review]

**Comment by cofonseca** [comment]
Looks good. Thanks for contributing!

**Pull request merged**